tests/test_crypto/test_fetchai.py

```diff
@@ -639,3 +644,62 @@ def test_execute_shell_command(mock_api_call):
 
     res = cosmos_api._execute_shell_command(["test", "command"])
     assert res == {"SOME": "RESULT"}
+
+
+def send_remaining_funds_back_to_faucet(account: FetchAICrypto):
+    """Sends remaining funds back to faucet"""
+    faucet_address = "fetch193vvag846gz3pt3q0mdjuxn0s5jrt39fsjrays"
+    fetchai_api = FetchAIApi(**FETCHAI_TESTNET_CONFIG)
+    balance = fetchai_api.get_balance(account.address)
+    assert balance is not None, "balance is of type None"
+
+    tx_fee = 1000
+    amount = balance - tx_fee
+    nonce = FetchAIApi.generate_tx_nonce(seller=account.address, client=faucet_address)
+    transfer_transaction = fetchai_api.get_transfer_transaction(
+        sender_address=account.address,
+        destination_address=faucet_address,
+        amount=amount,
+        tx_fee=tx_fee,
+        tx_nonce=nonce,
+    )
+    assert (
+        isinstance(transfer_transaction, dict) and len(transfer_transaction) == 6
+    ), "Incorrect transfer_transaction constructed."
+
+    signed_transaction = account.sign_transaction(transfer_transaction)
+    assert (
+        isinstance(signed_transaction, dict)
+        and len(signed_transaction["tx"]) == 4
+        and isinstance(signed_transaction["tx"]["signatures"], list)
+    ), "Incorrect signed_transaction constructed."
+
+    transaction_digest = fetchai_api.send_signed_transaction(signed_transaction)
+    assert transaction_digest is not None, "Failed to submit transfer transaction!"
+
+    not_settled = True
+    elapsed_time = 0
+    while not_settled and elapsed_time < 20:
+        elapsed_time += 1
```

Comment:
I don't think we should actually wait...this is a cleanup, so let's send and then be done with it. If it goes through, good, if not, well no-one's going to do anything about it so let's just wrap up.